crates/hotshot/src/traits/networking/combined_network.rs

```diff
@@ -106,7 +108,7 @@ impl<TYPES: NodeType> CombinedNetworks<TYPES> {
 
         Self {
             networks,
-            message_cache: Arc::new(RwLock::new(LruCache::new(
+            message_cache: Arc::new(PlRwLock::new(LruCache::new(
```

Comment:
Why the switch to parking lot library?

Reply:
The original was an async lock, which is slower and we would really only need if we were holding across `.await` points. 

`parking_lot` is kind of std++ when it comes to `RwLocks/Mutexes`, it performs better in most cases and is my goto for a sync lock